### ISSUE TITLE
Project review by Albertas Dvirnas

### ISSUE THREAD

**Issue by albertasdvirnas** [teokem/project-work-2018-KlasL#1]
Thank you for sharing the project, here are my comments (mostly from the critical side), on it.

Documentation:
- the project is missing instructions on how to run the notebook.
- it would be good to change the environment name into something more meaningful that LUcompute, which we used for the exercises
- I've ran into an error while loading the 'data/smhi-opendata_19_20_63050_20190103_115638.csv' file, maybe your operating system does not differentiate between lowercase and uper case? Just change data to Data
- There are some uncommented cells which make it harder to understand the purpose of the tables, and for example output 16 just outputs the same as 14, except that there's Kelvin's instead of Celcius, so it's basically the same table.
- "The algorithm from Hempel et al.2013" - here you could perhaps use markdown to cite the paper properly
- There is a typo in formula for k
Output:
- The processed data should be saved to disk using pandas
- The figure is saved, but should be publication ready (7x7) is not the recommended dimensions, and y-label of second subplot overlaps with the first plot, and the legends go beyond the frames of plots.
- There is no DOI, but I guess that can be added after some corrections

So to summarize there are a few points that should be addressed (mainly instructions and the folder name, doi). Apart from that it would do with more plots to make the project more vivid for the reader!
